.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+
+jobs:
+    Test:
+        docker:
+            - image: kkiesling/dagmc_stats_depends:0.2
```

Comment:
I will eventually move this image to the Svalinn docker hub when this gets reviewed